review_checker_desktop/review_checker_desktop.model.lkml

```diff
@@ -0,0 +1,9 @@
+connection: "telemetry"
+label: "Review Checker Desktop"
+# Include files from looker-hub or spoke-default below. For example:
+# include: "//looker-hub/review_checker_desktop/explores/*"
+# include: "//looker-hub/review_checker_desktop/dashboards/*"
+include: "//looker-hub/review_checker_desktop/views/*"
+# include: "views/*"
+# include: "explores/*"
```

Comment:
@betling FYI if you go the typical route of defining refinements of the views and new explores here in looker-spoke-default, then at that point you'd need to uncomment these two lines so the model includes the associated view & explore files.

Reply:
great, thanks for your help @sean-rose !